ci/environment.yml

```diff
@@ -1,22 +1,27 @@
 name: uxarray_build
 channels:
   - conda-forge
+  - nodefaults
 dependencies:
+  - antimeridian
   - cartopy
   - dask
   - datashader
   - gmpy2
+  - h5netcdf
+  - hdf5
   - holoviews
   - matplotlib-base
   - matplotlib-inline
   - netcdf4
   - numba
-  - numpy
+  - numpy<2
```

Comment:
2.0 was just released, any particular reason this won't be compatible with the latest versions going forward.

Reply:
We had some isolated, but reproducible issues with the most recent release for some of our visualizations tasks. Our Conda and PyPi releases have it pinned, so this updates it since I left that out last release. 

https://github.com/UXARRAY/uxarray/blob/3b27d3f3ccf1d4a84a28f6538b2a91f79e051160/pyproject.toml#L36